### PR TITLE
Improvement for Statement.create and fix for issue #9

### PR DESCRIPTION
**Pull request by MarcHauptmann**
`Statement.create` throws `IllegalArgumentException` when invalid statement is created by rated author.

To fix issue #9 I replaced the `List` for tags by a `Set`. This also eases dealing with tags since their order is no longer important.

### CODE REVIEW

File: app/models/Statement.scala [hunk 1]
```diff
@@ -170,12 +170,17 @@ object Statement {
 	}
 
 	def create(implicit connection: java.sql.Connection, title: String, author: Author, cat: Category, quote: Option[String], quote_src: Option[String], rating: Option[Rating], merged_id: Option[Long]): Statement = {
+    if(author.rated) {
+      val message: String = "a statement with rated author must be rated and and not merged"
+
```
Comment (stheophil) [inline app/models/Statement.scala:181]: merged_id is a bad name I think. It should be called linked_id. Please change the error msg to say the statement must not be "linked"

File: app/models/Statement.scala [hunk 2]
```diff
@@ -318,7 +323,7 @@ object Statement {
 					quote, quote_src,
 					List[Entry](),
 					latestEntry,
-					(tag_id, tag_name, tag_important).zipped.map( (id, name, important) => Tag(id, name, important) ).toList.sortBy(_.name),
+					(tag_id, tag_name, tag_important).zipped.map( (id, name, important) => Tag(id, name, important) ).toSet,
```
Comment (stheophil) [inline app/models/Statement.scala:332]: Should we define an implicit Ordering for Tags? They should be displayed ordered by name I think.

Comment (MarcHauptmann) [inline app/models/Statement.scala:332]: That would make sense. I'll implement that.

Comment (stheophil) [inline app/models/Statement.scala:332]: Did that already in the latest commit.

**Pull request merged**